### PR TITLE
Add some funcs and vtable names

### PR DESCRIPTION
**Pull request by wolfcomp**
Added most classes for public content and moved std::vector to be a class definition
Fixed a case where there was a double define of a dtor

### CODE REVIEW

**Comment by Haselnussbomber** [comment]
Couple times you mispelled `PuclicContent`.

File: ida/data.yml [hunk 1]
```diff
@@ -10089,12 +10092,90 @@ classes:
...
+  Client::Game::InstanceContent::PublicContentDelubrumSavage:
+    vtbls:
+      - ea: 0x141C3A280
+        base: Client::Game::InstanceContent::PublicContentDirector
+    funcs:
+      0x1414B9740: ctor
+  Client::Game::InstanceContent::PuclicContentZadnor:
+    vtbls:
+      - ea: 0x141C37F78
+        base: Client::Game::InstanceContent::PuclicContentBozja
```
Comment (Haselnussbomber) [inline ida/data.yml:10164]: Puclic -> Public, now as suggestion.^^

```suggestion
  Client::Game::InstanceContent::PublicContentBozja:
    vtbls:
      - ea: 0x141C36828
        base: Client::Game::InstanceContent::PublicContentDirector
    funcs:
      0x1414B8A00: ctor
  Client::Game::InstanceContent::PublicContentSoutherFront:
    vtbls:
      - ea: 0x141C373D0
        base: Client::Game::InstanceContent::PublicContentBozja
    funcs:
      0x1414B9210: ctor
  Client::Game::InstanceContent::PublicContentDelubrum:
    vtbls:
      - ea: 0x141C396D0
        base: Client::Game::InstanceContent::PublicContentDirector
    funcs:
      0x1414B96B0: ctor
  Client::Game::InstanceContent::PublicContentDelubrumSavage:
    vtbls:
      - ea: 0x141C3A280
        base: Client::Game::InstanceContent::PublicContentDirector
    funcs:
      0x1414B9740: ctor
  Client::Game::InstanceContent::PublicContentZadnor:
    vtbls:
      - ea: 0x141C37F78
        base: Client::Game::InstanceContent::PublicContentBozja
```

**Review by Haselnussbomber — CHANGES_REQUESTED** [review]

File: ida/data.yml [hunk 2]
```diff
@@ -318,6 +316,11 @@ functions:
   0x14192E720: CollisionFree
 
 classes:
+  std::vector:
+    funcs:
+      0x1415f1A90: dtor
+      0x140335660: SetSize
```
Comment (Soreepeong) [inline ida/data.yml:322]: `std::vector<AtkValue>::resize(size_t)`

File: ida/data.yml [hunk 3]
```diff
@@ -318,6 +316,11 @@ functions:
...
+      0x140921020: _Buy_raw # according to kizer based on source code for stl
```
Comment (Soreepeong) [inline ida/data.yml:323]: `std::vector<T>::_Buy_raw(size_t)` where `sizeof(T)` is 8.

File: ida/data.yml [hunk 4]
```diff
@@ -318,6 +316,11 @@ functions:
   0x14192E720: CollisionFree
 
 classes:
+  std::vector:
+    funcs:
+      0x1415f1A90: dtor
```
Comment (Soreepeong) [inline ida/data.yml:321]: [`std::vector<T>::_Tidy()`](https://github.com/microsoft/STL/blob/3eac329d1f614ecf138d96c22a3b02f87076bc4a/stl/inc/vector#L2039) where [`std::is_trivially_destructible_v<T>`](https://en.cppreference.com/w/cpp/types/is_destructible) [is `true`](https://github.com/microsoft/STL/blob/3eac329d1f614ecf138d96c22a3b02f87076bc4a/stl/inc/xmemory#L1110C1-L1129).

File: ida/data.yml [hunk 5]
```diff
@@ -318,6 +316,11 @@ functions:
   0x14192E720: CollisionFree
 
 classes:
+  std::vector:
```
Comment (Soreepeong) [inline ida/data.yml:319]: It might not be a good idea to have something called `std::vector` when there exist many variations on `<T>` part.

**Review by Soreepeong — COMMENTED** [review]
Multiple instances of `std::vector<T>` should be made, as the differences in `T` does matter.

**Comment by wolfcomp** [comment]
Closing as part of it was merged with #778

**Pull request closed without merging**